src/Vezeeta.Domain/Entities/Request.cs

```diff
@@ -0,0 +1,32 @@
+﻿
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Vezeeta.Domain.Enums;
+
+namespace Vezeeta.Domain.Entities
+{
+    public class Request
```

Comment:
For all entities, you should be inheriting from `BaseEntity<T>` where T is the datatype of the ID for your entity.